PWGHF/D2H/Tasks/taskSigmacToCascade.cxx

```diff
@@ -0,0 +1,203 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskSigmacToCascade.cxx
+///
+/// \note Task for Σc0,++ → Λc(→ K0sP) + π-,+ analysis
+/// \note Here the Lc from the cascade channel is obtained using the task taskLcToK0sP.cxx
+/// \author Rutuparna Rath <rrath@cern.ch>, INFN BOLOGNA and GSI Darmstadt
+/// In collaboration with Andrea Alici <aalici@cern.ch>, INFN BOLOGNA
+
+#include "CommonConstants/PhysicsConstants.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/TrackSelectionDefaults.h"
+
+#include "PWGHF/Core/HfHelper.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfTaskSigmacInCascade {
+
+  Configurable<bool> enableTHn{"enableTHn", false, "enable the usage of THn for Σc0, Σc++  and Σc0,++"};
+  Configurable<int> selectionFlagLcToK0sP{"selectionFlagLcToK0sP", 1, "Selection Flag for Lc"};
+  Configurable<int> selectionFlagLcbarToK0sP{"selectionFlagLcbarToK0sP", 1, "Selection Flag for Lcbar"};
```

Comment:
this info here is not used anywhere, neither in a `Filter` or checking candidate by candidate in the loop if these cuts from selector are applied. Please implement this, otherwise your combinatorics can explode for no reason.

Reply:
Thank you. Done.